Tests/iaas/entropy/entropy-check.py

```diff
@@ -42,7 +42,7 @@
 FLAVOR_ATTRIBUTES = {
     # https://docs.openstack.org/nova/2023.1/configuration/extra-specs.html#hw-rng
     # type: bool
-    "hw_rng:allowed": True,
+    "hw_rng:allowed": "True",  # testing showed that it is indeed a string?
```

Comment:
Well, what I tried to do here (and apparently failed to achieve) is to express the sheer surprise at this fact, because it contradicts the documentation. I could change it to `?!`

Reply:
it's fine :slightly_smiling_face: